src/vue/components/word-usage-example-item.vue

```diff
@@ -13,9 +13,9 @@
             <source-icon class="alpheios_word_usage_list_item__source-link-svg"></source-icon>
           </span>
         </div>
-        <div class="alpheios_word_usage_list_item__text_prefix" v-html="prefixHtml"></div>
+        <div class="alpheios_word_usage_list_item__text_prefix" v-html="prefixHtml" @dblclick="selectAWord"></div>
```

Comment:
why is it necessary to add a separate handler here? Doesn't main Alpheios event handler pick up the event?

Reply:
no - in embed-lib it is not working
I am not sure that it was ever working there